collect.py

```diff
@@ -214,6 +226,7 @@ def run_test(bug, browser, driver, op_sequence=None):
         print('Continuing...')
 
     screenshot(driver, 'data/%d_%s.png' % (bug['id'], browser))
```

Comment:
Let's add a new function that calls the functions to take a screenshot, store the page source and store the DOM tree.